page/partials/advisors.ejs

```diff
@@ -0,0 +1,82 @@
+<div class="container advisors">
+
+        <div class="row">
+            <div class="col-md-5">
+                <div class="has-carousel lazyLoad">
+                    <div>
+                        <h3>Advisors</h3>
+                        <p>"The tokenization of the equity financing could 
+                        be a major breakthrough in a capital market
+                        industry that has not seen any major change
+                        over the last 30 years (since the move from the
+                        open out cry / pit tranding to electronic
+                            trading)."</p>
+                        <div class="signature">
+                            <h3>Jean-Michel Pailhon</h3>
+                            <h6>Ledger</h6>
+                        </div>
+                    </div> <!--end advisor -->
+                    <div>
+                        <h3>Advisors</h3>
+                        <p>"The tokenization of the equity financing could 
+                        be a major breakthrough in a capital market
+                        industry that has not seen any major change
+                        over the last 30 years (since the move from the
+                        open out cry / pit tranding to electronic
+                            trading)."</p>
+                        <div class="signature">
+                            <h3>Jean-Michel Pailhon</h3>
+                            <h6>Ledger</h6>
+                        </div>
+                    </div> <!--end advisor -->
+
+
+                </div> <!-- end carousel -->
+                    
+            </div>
+            <div class="col-md-7">
+
+                <div class="row person-info-container">
+                    <div class="col-md-6">
+                        <div class="person-info">
+                            <img class="rounded-image" src="../assets/mostafabalatabw.jpg"/>
+                            <div class="person-details">
+                                <h4 class="name"><a href="#" >Micheal Jackson</a></h4>
+                                <h4 >Mangrove Capital</h4>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-6">
+                        <div class="person-info">
+                            <img class="rounded-image" src="../assets/mostafabalatabw.jpg"/>
+                            <div class="person-details">
+                                <h4 class="name"><a href="#" >Micheal Jackson</a></h4>
+                                <h4 >Mangrove Capital</h4>
+                            </div>
+                        </div>
+                    </div>
+                </div><!-- End row -->        
```

Comment:
do not create new rows. Just put all images in one row and let css figure out when to break to new line. Then it's responsive.  Read: https://stackoverflow.com/a/31049088/580181